src/plugins/csvstorage/csvstorage.c

```diff
@@ -0,0 +1,350 @@
+/**
+ * \file
+ *
+ * \brief Source for csvstorage plugin
+ *
+ * \copyright BSD License (see doc/COPYING or http://www.libelektra.org)
+ *
+ */
+
+
+#ifndef HAVE_KDBCONFIG
+# include "kdbconfig.h"
+#endif
+
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <kdberrors.h>
+
+#include "csvstorage.h"
+
+static char *parseLine(char *_line, char delim, unsigned long offset)
```

Comment:
Why `_line` and not `line`?
Its a bit nitpicking, but basically names started with `_` are reserved and should not be used.